tests/e2e/backup_restore_suite_test.go

```diff
@@ -97,6 +97,10 @@ func runApplicationBackupAndRestore(brCase ApplicationBackupRestoreCase, expecte
 	// create DPA
 	backupName, restoreName := prepareBackupAndRestore(brCase.BackupRestoreCase, updateLastInstallTime)
 
+	// Ensure that an existing backup repository is deleted
```

Comment:
would add this new logic inside this function  https://github.com/openshift/oadp-operator/blob/68a1c47edd28a9acfd5f93bb76dd08989a6667ac/tests/e2e/backup_restore_suite_test.go#L54

this way, avoids duplication, by needing to add it to 2 files

`tests/e2e/virt_backup_restore_suite_test.go` also uses this function https://github.com/openshift/oadp-operator/blob/68a1c47edd28a9acfd5f93bb76dd08989a6667ac/tests/e2e/virt_backup_restore_suite_test.go#L86

Reply:
this seems like a trivial change and would make the test suites less clear and less uncoupled.